lib/getEnrollments.js

```diff
@@ -3,8 +3,8 @@ const canvas = require('./Canvas')
 const kopps = require('./Kopps')
 const ug = require('./Ug')
 
-// Written as constant to avoid mistypes
-const ANTAGNA = 'Admitted not registered'
+// Written role replaced with account role: 'Admitted not registered' -> role_id: 25
```

Comment:
Suggestion (quite irrelevant maybe): this comment is going to be read in the code without the git history. I think is a nice idea that you explain what "25" means but try to say it without referencing the past

Short version: don't use "replace" there :)

Otherwise you can remove the comment since the name of the variable you chose "antagnaRoleId" is quite self-explanatory!